src/main/java/org/dflib/jjava/Extension.java

```diff
@@ -0,0 +1,6 @@
+package org.dflib.jjava;
+
+public interface Extension {
```

Comment:
Would be great to have some javadoc for this Interaface and it's main method, as this is intended as a public API